app/javascript/components/Payment/Payment.js

```diff
@@ -269,6 +269,12 @@ export class Payment extends Component {
       window.fbq('track', 'AddPaymentInfo', eventPayload);
     }
 
+    console.log(
+      'ABOUT TO EMIT EVENT WITH: ',
+      eventPayload,
+      this.props.formData
+    );
```

Comment:
Do you still want this here?

Reply:
Haha no, oups... I've removed it locally but need to push. There's another change I also made to the event log file. I'm thinking of incorporating a couple bug fixes onto this branch as well so maybe this will be a kind of a WIP review after all